src/schema.ts

```diff
@@ -5,23 +5,42 @@ export const typeDefs = `
       hello: String
       user(data: QueryUserInput!): User
       users(data: QueryUsersInput = { skippedUsers: 0, maxUsers: ${MAX_USERS} }): UsersResponse
+    }              
+    
+    type Address {
+      id: ID!
+      cep: String!
+      street: String!
+      streetNumber: String!
+      complement: String
+      neighborhood: String!
+      city: String!
+      state: String!
     }
-      
+
+    type SimplifiedUser {
+      id: ID!
+      name: String!
+      email: String!
+      birthDate: String!
+    }
```

Comment:
Não acho que seja necessário esse tipo. No GraphQL o client pode escolher os campos que ele quer que retorne, então se vc mantém tudo retornando um User (com todos os dados), ele poderia escolher entre retornar tudo ou apenas essas campos de qualquer forma.

Reply:
Eu entendi, pelo enunciado da issue, que não deveria haver a opção de retornar os dados de addresses na mutation de login, apenas nas queries user e users, por isso criei o esse tipo simplificado, sem os addresses.
"...we're going to update the two queries: "user" and "users". Now they should also return the addresses."
Se a ideia é que essa mudança também impacte o login, concordo que o tipo é desnecessário e basta seguir retornando User, como pontuado.

Reply:
Falar que é pra atualizar as 2 queries não automaticamente obriga o login a ficar como está. Mas entendi o ponto. A ideia no onboard é não ter com muitas minúcias de requisitos, ou seja, soluções mais simples em termos de escopo.